web-ui/src/main/resources/catalog/views/default/directives/partials/mdactionmenu.html

```diff
@@ -154,6 +154,17 @@
           <span data-translate="">enableWorkflow</span>
         </a>
       </li>
+      <!--metadata workflow assistance external link(s)-->
+      <li role="menuitem"
+          data-ng-if="isMdWorkflowEnable && md.isWorkflowEnabled()
+          && isMdWorkflowAssistEnable && md.isOwned() && user.isUserAdminOrMore()" data-ng-repeat="app in workflowAssistApps">
+        <a href=""
```

Comment:
Please add `rel="noopener"` to <a> attribute to fix potential security issue.

Reply:
This is added. Also the {parameter} is separated to {uuid} and {lang} to be more readable.

Here is the end result:
![image](https://user-images.githubusercontent.com/74916635/158441482-8bdd5abc-f9a7-4e31-bd63-fdd90275918c.png)